### DISCUSSION TITLE
Working with DOMA and Xamarin/.NET Multi-platform App UI (.NET MAUI)....

### DISCUSSION THREAD

**Discussion by gatorpower** [domaframework/doma#845] [Q&A]
I am setting up a DEV environment and I was asked if DOMA could be used to process standard database requests, but allow developers who are more comfortable in Xamarin (C#) to build in the framework.   I know that Xamarin will be deprecated soon, if it has not already, and the switch will be to .NET MAUI.

Is this a thing?  If so, how would one go about using it?  Thanks for any help.

**Comment by nakamura-to** [comment]
There is no exceptional support for Xamarin developers. Doma is just a database access library for Java.
